test_runner/regress/test_tenant_detach.py

```diff
@@ -750,6 +751,7 @@ def test_metrics_while_ignoring_broken_tenant_and_reloading(
     env.pageserver.allowed_errors.append(
         r".* Changing Active tenant to Broken state, reason: broken from test"
     )
+    env.pageserver.allowed_errors.append(r".*Dropped remote consistent LSN updates for tenant *.")
```

Comment:
```suggestion
    env.pageserver.allowed_errors.extend(
        [
            r".* Changing Active tenant to Broken state, reason: broken from test",
            r".*Dropped remote consistent LSN updates for tenant *.",
        ]
    )
```

Reply:
I had tried this earlier but the linter complains about this

Reply:
Do you mean https://github.com/neondatabase/neon/pull/6273/commits/6e22998da83827a86171ccc3d4d7c4d6d5dcd701?

You've used `append` instead of `extend` there